src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -76,96 +82,103 @@
   public static final String LOBBY_GAME_RECONNECTION = "triplea.lobby.game.reconnection";
   public static final String TRIPLEA_ENGINE_VERSION_BIN = "triplea.engine.version.bin";
-  public static final String PROXY_HOST = "proxy.host";
-  public static final String PROXY_PORT = "proxy.port";
-  public static final String TRIPLEA_DO_NOT_CHECK_FOR_UPDATES = "triplea.doNotCheckForUpdates";
-  // has the memory been manually set or not?
-  public static final String TRIPLEA_MEMORY_SET = "triplea.memory.set";
+  private static final String TRIPLEA_DO_NOT_CHECK_FOR_UPDATES = "triplea.doNotCheckForUpdates";
+
   public static final String TRIPLEA_SERVER_START_GAME_SYNC_WAIT_TIME = "triplea.server.startGameSyncWaitTime";
   public static final String TRIPLEA_SERVER_OBSERVER_JOIN_WAIT_TIME = "triplea.server.observerJoinWaitTime";
   // non-commandline-argument-properties (for preferences)
   // first time we've run this version of triplea?
-  private static final String TRIPLEA_FIRST_TIME_THIS_VERSION_PROPERTY =
-      "triplea.firstTimeThisVersion" + ClientContext.engineVersion();
-  private static final String TRIPLEA_LAST_CHECK_FOR_ENGINE_UPDATE = "triplea.lastCheckForEngineUpdate";
-  // only for Online?
-  public static final String TRIPLEA_MEMORY_ONLINE_ONLY = "triplea.memory.onlineOnly";
-  // what should our xmx be approximately?
-  public static final String TRIPLEA_MEMORY_XMX = "triplea.memory.Xmx";
-  public static final String TRIPLEA_MEMORY_USE_DEFAULT = "triplea.memory.useDefault";
-  public static final String SYSTEM_INI = "system.ini";
+  private static final String SYSTEM_INI = "system.ini";
   public static final int MINIMUM_CLIENT_GAMEDATA_LOAD_GRACE_TIME = 20;
-  public static final int DEFAULT_CLIENT_GAMEDATA_LOAD_GRACE_TIME =
+  private static final int DEFAULT_CLIENT_GAMEDATA_LOAD_GRACE_TIME =
       Math.max(MINIMUM_CLIENT_GAMEDATA_LOAD_GRACE_TIME, 25);
   // need time for network transmission of a large game data
   public static final int MINIMUM_SERVER_OBSERVER_JOIN_WAIT_TIME = MINIMUM_CLIENT_GAMEDATA_LOAD_GRACE_TIME + 10;
-  public static final int DEFAULT_SERVER_OBSERVER_JOIN_WAIT_TIME =
+  private static final int DEFAULT_SERVER_OBSERVER_JOIN_WAIT_TIME =
       Math.max(DEFAULT_CLIENT_GAMEDATA_LOAD_GRACE_TIME + 10, 35);
   public static final int ADDITIONAL_SERVER_ERROR_DISCONNECTION_WAIT_TIME = 10;
   public static final int MINIMUM_SERVER_START_GAME_SYNC_WAIT_TIME =
       MINIMUM_SERVER_OBSERVER_JOIN_WAIT_TIME + ADDITIONAL_SERVER_ERROR_DISCONNECTION_WAIT_TIME + 110;
-  public static final int DEFAULT_SERVER_START_GAME_SYNC_WAIT_TIME =
+  private static final int DEFAULT_SERVER_START_GAME_SYNC_WAIT_TIME =
       Math.max(Math.max(MINIMUM_SERVER_START_GAME_SYNC_WAIT_TIME, 900),
           DEFAULT_SERVER_OBSERVER_JOIN_WAIT_TIME + ADDITIONAL_SERVER_ERROR_DISCONNECTION_WAIT_TIME + 110);
 
   private static final String MAP_FOLDER = "mapFolder";
 
+  private static String[] COMMAND_LINE_ARGS =
```

Comment:
Is this array used to determine if a given String is a valid argument?
If so, wouldn't it be easier to use a collection instead of an array? ('COMMAND_LINE_ARGS.contains("THIS STRING");')